### DISCUSSION TITLE
Mac OSX said damaged

### DISCUSSION THREAD

**Discussion by SaveMyBlackberries** [stakira/OpenUtau#639] [Q&A]
Hi, 
Thanks for great work.
Now I run the openUTAU in Mac OSX 10.15.7. It opens the warning with "OpenUtau is damaged and can't be opened. you should move it to the trash."
Version is 0.1.51
Is there any tips? Do I have to update to newest system? Cause I have some main works relies on this OSX version.
Thanks a lot!

**Comment by pseudo-producer** [comment]
Have you tested the current version (0.1.52)?
